hoomd/mpcd/SphereGeometry.h

```diff
@@ -0,0 +1,195 @@
+
+#ifndef MPCD_SPHERE_GEOMETRY_H_
+#define MPCD_SPHERE_GEOMETRY_H_
+
+#include <math.h>
```

Comment:
Prefer C++ headers
```suggestion
#include <cmath>
```
You should also move this down before the `#ifdef NVCC` to keep all the standard library includes together.